tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -32,6 +32,10 @@ internal static class Resources
         public const string AspNetCoreProcessNotFound = "Could not find the ASP.NET Core applicative process.";
         public const string VersionConflict = "Tracer version 1.x can't be loaded simultaneously with other versions and will produce orphaned traces. Make sure to synchronize the Datadog.Trace NuGet version with the installed automatic instrumentation package version.";
 
+        public static string ProfilerVersion(string version) => $"The native library version {version} is loaded into the process.";
+
+        public static string TracerVersion(string version) => $"The tracer version {version} is loaded into the process.";
```

Comment:
Should we use "managed library" to be consistent with "native library". Or, to be more precise
- native tracer library
- managed tracer library

Reply:
I don't mind either way. We can do the change in another PR if you feel strongly about it 🙂 (this one has been rebased too many times already 😅 )